tests/e2e/support/commands.js

```diff
@@ -2,6 +2,18 @@
 // This command logs you in as either a GP (General Participant), PF (Participating Family) or Admin
 // ** Right now just logs you in as an example user **
 Cypress.Commands.add("login", () => {
+    const user = {
```

Comment:
Could you write this to be specific to the type of user, or configurable with an argument?

Reply:
I do not have any other credentials so I can only login as PF